test/myFTClient.spec.js

```diff
@@ -835,172 +835,197 @@ describe('myFT Client proxy', () => {
 		});
 	});
 
-		//new orgainic v3/kat methods
-		describe('Followed_by_Kat concepts', () => {
-			const relProps = Object.assign({}, myFT.followedProperties, {byTool: 'myFTClient.spec', isTest: true});
-			const {ids, subjects} = require('./mocks/fixtures/userFollowsConceptByKat');
-			const followedByKatRes = require('./mocks/fixtures/followByKatResp');
+	//v3/kat remove users endpoint
+	describe('Remove user from a licence', () => {
 
-			afterEach(done => {
-					nock.cleanAll();
-					done();
-			});
+		afterEach(done => {
+			nock.cleanAll();
+			done();
+		});
 
-			it('Should set concept(s) follows by a user ', done => {
-					nock(baseUrl)
-						.post('/kat/user/follows')
-						.query(true)
-						.reply(200, () => followedByKatRes);
+		it('should return 204 with empty object when calling /kat/user/remove remove all of a users relationships related to a given licence', () => {
```

Comment:
Not 100% sure this is testing anything that the function actually does itself. The response returned is based on whatever `nock` returns.
Also, we don't receive an empty object, as `resp` contains the `status` property.